### ISSUE TITLE
Content to be added

### ISSUE THREAD

**Issue by rolandlo** [xournalpp/xournalpp.github.io#21]
I made some thoughts about what needs to be added content-wise to make the website (more or less) complete. Feel free to add/remove and restructure. The + stands for "has to be added from scratch".

- [ ] **Overview** (+typical workflow, +overview over content of other sections)
- [ ] **Tools & Editing** (+text, +font/font size, +images, + tool switching including floating toolbox and tool cycle plugin, + vertical space, +colors, +fill, +opacity, +line width, +pressure sensitivity)
- [ ] **Configuration** (+preferences, +page default, +toolbar customization, +colors, +setting up device classes, +setting up stylus and mouse buttons)
- [ ] **Tips** (+some useful tips of how to use Xournal++ efficiently)
- [ ] +**Selection & Snapping** (rectangle, region and object select, snapping to grid, rotation snapping, modifiers, explain moving, scaling and rotation with snapping turned on)
- [ ] +**Pages & Layers** (sidebar, inserting, deleting, backgrounds)
- [ ] +**Layout, Zoom & Navigation** (managing different layouts, zoom modes, zoom steps, navigation with keyboard, hand tool, toolbar, accelerators)
- [ ] +**File operations** (load, annotate pdf, attached mode, export as pdf, export in various formats, export options)
- [ ] +**Audio recording** (how to record, playback, pause, select play object, set audio folder and options)
- [ ] +**Getting help** (website, issues, disussions, gitter)

**Comment by Technius** [comment]
One idea I had was guides for specific use cases, e.g.

* [ ] +**Using Xournal++ for homework** (filling out blanks, inserting images, inserting LaTeX, etc.)
* [ ] +**Using Xournal++ for note-taking** (workflow tips?)
* [ ] +**Tips for Educators** (how to present, workflow for grading, turning lecture "slides" into notes, etc.)

**Comment by rolandlo** [comment]
> ...or maybe these are better left as blog posts?

I think adding blog posts is a great idea. We could use [this plugin](https://pypi.org/project/mkdocs-blog-plugin/) for that. We have some enthusiastic users who may like to share their personal tips and recommendations in a blog post.

**Comment by arjun-krishna1** [comment]
I just created a pull request that updated the Overview section

**Comment by Fatih20** [comment]
> I made some thoughts about what needs to be added content-wise to make the website (more or less) complete. Feel free to add/remove and restructure. The + stands for "has to be added from scratch".
> 
> * [ ]  **Overview** (+typical workflow, +overview over content of other sections)
> * [ ]  **Tools & Editing** (+text, +font/font size, +images, + tool switching including floating toolbox and tool cycle plugin, + vertical space, +colors, +fill, +opacity, +line width, +pressure sensitivity)
> * [ ]  **Configuration** (+preferences, +page default, +toolbar customization, +colors, +setting up device classes, +setting up stylus and mouse buttons)
> * [ ]  **Tips** (+some useful tips of how to use Xournal++ efficiently)
> * [x]  +**Selection & Snapping** (rectangle, region and object select, snapping to grid, rotation snapping, modifiers, explain moving, scaling and rotation with snapping turned on)
> * [ ]  +**Pages & Layers** (sidebar, inserting, deleting, backgrounds)
> * [ ]  +**Layout, Zoom & Navigation** (managing different layouts, zoom modes, zoom steps, navigation with keyboard, hand tool, toolbar, accelerators)
> * [ ]  +**File operations** (load, annotate pdf, attached mode, export as pdf, export in various formats, export options)
> * [ ]  +**Audio recording** (how to record, playback, pause, select play object, set audio folder and options)
> * [x]  +**Getting help** (website, issues, disussions, gitter)
> * [x]  +**File structure** (+where to find which files on what OS and with what version)

I assume most of those are going to be put in the user guide section? Putting that many information on the home would overwhelm new visitor.

**Comment by Fatih20** [comment]
> I made some thoughts about what needs to be added content-wise to make the website (more or less) complete. Feel free to add/remove and restructure. The + stands for "has to be added from scratch".
> 
> * [ ]  **Overview** (+typical workflow, +overview over content of other sections)
> * [ ]  **Tools & Editing** (+text, +font/font size, +images, + tool switching including floating toolbox and tool cycle plugin, + vertical space, +colors, +fill, +opacity, +line width, +pressure sensitivity)
> * [ ]  **Configuration** (+preferences, +page default, +toolbar customization, +colors, +setting up device classes, +setting up stylus and mouse buttons)
> * [ ]  **Tips** (+some useful tips of how to use Xournal++ efficiently)
> * [x]  +**Selection & Snapping** (rectangle, region and object select, snapping to grid, rotation snapping, modifiers, explain moving, scaling and rotation with snapping turned on)
> * [ ]  +**Pages & Layers** (sidebar, inserting, deleting, backgrounds)
> * [ ]  +**Layout, Zoom & Navigation** (managing different layouts, zoom modes, zoom steps, navigation with keyboard, hand tool, toolbar, accelerators)
> * [ ]  +**File operations** (load, annotate pdf, attached mode, export as pdf, export in various formats, export options)
> * [ ]  +**Audio recording** (how to record, playback, pause, select play object, set audio folder and options)
> * [x]  +**Getting help** (website, issues, disussions, gitter)
> * [x]  +**File structure** (+where to find which files on what OS and with what version)

I'm interested in completing these. How are they structured though? Is every section explaining the apps individual functionality has a GIF accompanied with an explanation of what is happening?

**Comment by Technius** [comment]
As I understand it, these pages are supposed to be detailed, in-depth explanations of the specific features that they are about. A general overview should be provided in the Overview page and in the whirlwind tour (#40).

**Comment by Fatih20** [comment]
> As I understand it, these pages are supposed to be detailed, in-depth explanations of the specific features that they are about. A general overview should be provided in the Overview page and in the whirlwind tour (#40).

Should these sections have pictures embedded in them? After reading some of the tools sections that have been completed, I think some could be simplified and be more digestible if pictures (or moving pictures) are included.

**Comment by Technius** [comment]
> After reading some of the tools sections that have been completed, I think some could be simplified and be more digestible if pictures (or moving pictures) are included.

Yes, they definitely should have icons of the tools as well as pictures that show what the tools can create.
